### PR TITLE
seems to work :shipit:

### PR DESCRIPTION
**Pull request by SomethingGeneric**
@not-my-segfault sanity check?

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+license=('GPL3')
+arch=('any')
+url="https://github.com/crystal-linux/jade-tui"
+source=("jade-tui" "locales")
+depends=('jade' 'gum' 'openssl')
+md5sums=('f8bfed0ea3f1b5e85138b0c8b4770e48'
+         'cba37f460dcf5f678043df39d6378c35')
+
+package() {
+    mkdir -p ${pkgdir}/usr/{share/jade-tui,bin}
```
Comment (mchalunderscore) [inline PKGBUILD]: we should ideally be using the `install` command to do this and `cp` and set perms all at once, I believe my new `pkg-warner` tool has an example of this in its PKGBUILD

**Review by mchalunderscore — CHANGES_REQUESTED** [review]

**Comment by SomethingGeneric** [comment]
@not-my-segfault does this syntax for `install` look correct?

**Comment by mchalunderscore** [comment]
`-D -m` can be shortened to `-Dm` but otherwise this is perfect 👍

**Review by mchalunderscore — APPROVED** [review]

**Comment by mchalunderscore** [comment]
@axtloss Could you check this over so we can merge please? Thank you

**Comment by axtloss** [comment]
LGTM, @ShyyLexi would you say the current jade-tui can be merged into main?

**Comment by awa-vr** [comment]
Yea it's mostly still the same, just manual partition is not working yet

**Review by axtloss — APPROVED** [review]

**Pull request merged**